### PR TITLE
www.cookpad.com

### LINKED ISSUE #133750: www.cookpad.com
### Issue URL (Ads)
 [https://www.cookpad.com/recipe/5269023](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.cookpad.com%2Frecipe%2F5269023)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.4
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [x] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 Fix #133750 

### Add your comment and screenshots
<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt [hunk 1]
```diff
@@ -792,6 +792,8 @@ freeoffice.com##.nav-banner-top
 ||audee.jp/js/smart.banner/
 ada.com#?##page-wrap > main > aside[class]:has(> a[href^="https://app.adjust.com/"])
 homeworklib.com#?#.span4 > div.box:has(> a[href="https://www.homeworklib.com/links/homework-android-app"])
+cookpad.com#?#div > div:has( > a[href^="https://app.adjust.com/"])
```
Comment (zloyden) [inline AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:795]: Better write it like
```
cookpad.com#?#footer > div[class] > div[class]:has( > a[href^="https://app.adjust.com/"])
```

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -44,6 +44,7 @@ tokyo-sports.co.jp,soccerdigestweb.com,wiki.yjsnpi.nu,athleterecipe.com,javkimoc
 !
 !
 ||episodesex.org/banner/
+cookpad.com##div > div[data-ad-slot^="spweb-"]
```
Comment (zloyden) [inline JapaneseFilter/sections/specific.txt:47]: I don't see it on desktop, so you can move it to AdGuard Mobile Ads as
```
cookpad.com##div[data-ad-slot^="spweb-"]
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt:795]: I see. I will fix.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:47]: Sorry, I mistook that.

File: MobileFilter/sections/specific_web.txt [hunk 3]
```diff
@@ -1108,6 +1108,7 @@ pikabu.ru##.ads-under-post
 dailyportalz.jp##main > div > div.contents_title:first-child
 ||cpcdn.com/assets/device/ad/$domain=cookpad.com
 cookpad.com##.standard_ad_slot
+cookpad.com##div > div[data-ad-slot^="spweb-"]
```
Comment (zloyden) [inline MobileFilter/sections/specific_web.txt:1111]: As I can see, there is no difference whether to use `div >` or not , so you can remove this part.
```
cookpad.com##div[data-ad-slot^="spweb-"]
```

Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt:1111]: OK, I will fix.

**Review by zloyden — APPROVED** [review]

**Pull request merged**